easybuild/easyconfigs/a/ADMIXTURE/ADMIXTURE-1.3.0.eb

```
@@ -3,20 +3,23 @@ easyblock = 'PackedBinary'
 name = 'ADMIXTURE'
 version = '1.3.0'
 
-homepage = 'http://software.genetics.ucla.edu/admixture'
+homepage = 'https://dalexander.github.io/admixture/'
 description = """ADMIXTURE is a software tool for maximum likelihood estimation of individual ancestries from 
 multilocus SNP genotype datasets. It uses the same statistical model as STRUCTURE but calculates estimates much 
 more rapidly using a fast numerical optimization algorithm."""
 
 toolchain = SYSTEM
 
-source_urls = ['http://software.genetics.ucla.edu/%(namelower)s/binaries/']
+source_urls = ['https://dalexander.github.io/admixture/binaries/']
 sources = ['%(namelower)s_linux-%(version)s.tar.gz']
-checksums = ['41f209817a17981a717c9a4aa3d799da718ed080f3386e703927628c74ca6ca6']
+checksums = ['353e8b170c81f8d95946bf18bc78afda5d6bd32645b2a68658bd6781ff35703c']
+
+
+modextrapaths = {'PATH': 'admixture_linux-1.3.0'}
 
 sanity_check_paths = {
-    'files': ['admixture', 'admixture32'],
-    'dirs': []
+    'files': ['admixture_linux-1.3.0/admixture', 'admixture_linux-1.3.0/admixture32'],
+    'dirs': ['admixture_linux-1.3.0']
 }
```

We can add a sanity check command and also update the sanity check on files inside `bin`
```suggestion
sanity_check_paths = {
    'files': ['bin/admixture', 'bin/admixture32'],
    'dirs': []
}

sanity_check_commands = ["admixture --help"]
```